### PR TITLE
feat: add energy deposition test for OpticalTracker example

### PR DESCRIPTION
**Pull request by c-dilks**
BEGINRELEASENOTES
- Add test for average energy deposition in `OpticalTracker` example, for `Geant4SensitiveAction<Geant4OpticalTracker>`

ENDRELEASENOTES

### CODE REVIEW

**Comment by c-dilks** [comment]
This test should pass now, since #1067 is resolved.

**Comment by c-dilks** [comment]
Hmm... I'm not sure how to correctly handle units in the test. The `key4hep` job does not have access to `CLHEP` units, and at least in my local build, `dd4hep::eV != CLHEP::eV`.

File: examples/OpticalTracker/scripts/test_energy_deposition.C [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#include "CLHEP/Units/SystemOfUnits.h"
```
Comment (andresailer) [inline examples/OpticalTracker/scripts/test_energy_deposition.C:1]:
```suggestion
#include "DD4hep/DD4hepUnits.h"
```

Comment (andresailer) [reply examples/OpticalTracker/scripts/test_energy_deposition.C:1]: And dd4hep::ev etc

Comment (andresailer) [reply examples/OpticalTracker/scripts/test_energy_deposition.C:1]: Depending on if gean4 units are used or not one has to convert the energy in the file. Or convert the energy to the expected unit before it is written to file.

Comment (andresailer) [reply examples/OpticalTracker/scripts/test_energy_deposition.C:1]: Actually, here one should just use geant4 units in all cases.

**Comment by c-dilks** [comment]
Oh, thanks @andresailer! Sorry I got distracted working on other things...

**Pull request merged**